src/hooks/useUpdateEmoji.ts

```diff
@@ -61,7 +61,7 @@ export default function useUpdateEmoji() {
         ({ address }: { address: string }) =>
           address.toLowerCase() === accountAddress.toLowerCase()
       ) || {};
-    const maybeEmoji = label.split(' ')[0];
+    const maybeEmoji = label?.split(' ')[0] ?? '';
```

Comment:
since they're tied together by an index I wonder if using the accountColor value would be better since im p sure its an int

Reply:
no, it wouldn't because the amount of colors is smaller. 

Also, this change doesn't make sense,  because the label is always defined for wallet, no?